test/components/Vizr_test.js

```diff
@@ -10,16 +10,18 @@ import ExampleChart from '@/components/ExampleChart';
 import Chart from '@/components/Chart';
 import VizrVersion from '@/components/VizrVersion';
 
```

Comment:
I'm not sure how this test suite ever worked. When new charts are added, the Vizr component simply uses `push` to add them to the array, which alters this constant. Apparently only test ordering was keeping it from being a problem.